pkg/util/metrics/util_test.go

```diff
@@ -37,7 +37,7 @@ func TestRegisterMetricAndTrackRateLimiterUsage(t *testing.T) {
 		{
 			ownerName:   "owner_name",
 			rateLimiter: flowcontrol.NewTokenBucketRateLimiter(1, 1),
-			err:         "already registered",
```

Comment:
This should occur here no? The previous entry in the array is doing the exact same thing.

Reply:
Yes, the code has been refactored in this commit: https://github.com/kubernetes/kubernetes/commit/e8300dea529c45480596243385b3f484bc41eb3b, maybe used to avoid concurrent call.